jsapp/js/tos/tosForm.component.tsx

```diff
@@ -208,7 +215,7 @@ export default function TOSForm() {
           </h2>
 
           <AccountFieldsEditor
-            displayedFields={requiredFields}
+            displayedFields={fieldsToShow}
```

Comment:
So I tested this with various combinations of required fields, and the only problematic case was this:
```
  {
    "name": "organization",
    "required": false
  },
  {
    "name": "organization_type",
    "required": true
  },
  {
    "name": "organization_website",
    "required": false
  },
```

In such situation it's impossible to accept TOS, as `organization_type` doesn't appear. But as far as I remember, we agreed to not support this case, right? To add a comment somewhere about this

Reply:
Thanks for testing! I think this can be supported without breakage, if not I'll add a comment, possibly in the Constance help.
Xref https://github.com/kobotoolbox/kpi/pull/4747/files/69cbe70c622eecbcfa5546c1d08f396b9ef17e4b#r1423970261